### ISSUE TITLE
Document.create is expecting a method to be send

### ISSUE THREAD

**Issue by Jesusz0r** [Automattic/mongoose#9695]
Hi. Not a bug or feature request, just asking for help.
I have the following code 

```js
interface OrderDocument extends Document {
  userId: string;
  status: string;
  expiresAt: Date;
  ticket: TicketDocument;

  setCancelStatus(): Promise<OrderDocument>;
}

const orderSchema = new mongoose.Schema({
...
})

ordersSchema.methods = {
  setCancelStatus: async function (): Promise<OrderDocument> {
    this.set("status", Events.Status.OrderStatus.Cancelled);

    return this.save();
  },
};
```

As you can see, inside the `interface OrderDocument` I'm telling TypeScript that I have a method called `setCancelStatus` and afterwards I declare said method.
The problem is that when I try to create a new document using `Order.create` it expect me to send a `setCancelStatus` method as well as the other 4 properties.
Can you guys tell me what am I doing wrong when declaring the method inside the interface? Whats the proper way to declare it? Any documentation where I can check this out?

Thank you!

**Comment by vkarpov15** [comment]
This is an issue with our TypeScript bindings, fix will be in v5.11.8.

**Comment by Jesusz0r** [comment]
> This is an issue with our TypeScript bindings, fix will be in v5.11.8.

Thank you mate and sorry to bother you!
